src/blocks/star.css

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
You should manage the amount of active stars using modifier (stars--4, stars--3)